src/main/java/com/google/api/codegen/transformer/SampleTransformer.java

```diff
@@ -222,6 +224,88 @@ public void generateSamples(
     return methodSampleViews;
   }
 
+  // entry point for generating standalone samples using sample config.
```

Comment:
By "entry point" I meant that when generating samples using sample config (instead of gapic config), `LangGapicSamplesTransformer` should call this method rather than other overloads.